gnomad_hail/resources/import_resources.py

```diff
@@ -0,0 +1,82 @@
+from inspect import getmembers
+from typing import Dict, Tuple, Optional
+from gnomad_hail.resources.resource_utils import BaseResource, BaseVersionedResource
+import gnomad_hail.resources.grch37 as grch37
+import gnomad_hail.resources.grch38 as grch38
+import argparse
+
+
+# Generate a dictionary of resource available for import for a given genome build
+def get_module_importable_resources(module, prefix: Optional[str] = None) -> Dict[str, Tuple[str, BaseResource]]:
+    """
+    This takes a module that was imported and generates a list of all resources in this module that can be imported (i.e. with a path and import_func).
+    The dict produced is as follows:
+    keys: {prefix}.{resource_name}.{version} (with prefix only present if `prefix` is set, and `version` only present for versioned resources)
+    values: ({resource_name}[ version {version}], resource) with resource_name set to the variable name in the module and the version present for versioned resources.
+
+     The following example will generate a dict with all the resources in gnomad_hail.resources.grch37 that can be imported:
+
+    .. code-block:: python
+
+        import gnomad_hail.resources.grch37 as grch37
+        grch37_resources = get_module_importable_resources(grch37, prefix='grch37')
+    
+    :param module: Input module
+    :param prefix: 
+    :return: 
+    """
+    _prefix = f'{prefix}.' if prefix is not None else ''
```

Comment:
Nitpick, but just `if prefix` instead of `if prefix is not None` would be preferable here. Otherwise, passing an empty string for prefix would result in a prefix of ".".